packages/newspringchurchapp/src/prayer/PrayerPreviewCard/index.js

```diff
@@ -69,18 +47,9 @@ const StyledButton = styled({
 })(Button);
 
 const PrayerPreviewCard = withNavigation(
-  ({ imageSource, name, overlayColor, prayer, source, navigation, route }) => (
+  ({ overlayColor, navigation, route, ...props }) => (
     <>
-      <StyledCard>
-        <StyledCardContent>
-          <Avatar source={imageSource} />
-          <H3>Pray For {name}</H3>
-          <H6>{source}</H6>
-          <StyledBodyTextContainer>
-            <BodyText>{prayer}</BodyText>
-          </StyledBodyTextContainer>
-        </StyledCardContent>
-      </StyledCard>
+      <PrayerCard interactive={false} help={false} {...props} />
```

Comment:
The prop `help` is slightly hard to decipher. What does it do?

Reply:
I agree, it displays the helper text "how to pray" thing, what do you think a better name is?

Reply:
`showHelperText`? `showHelp`? `displayHelperText`? Pretty much anything that tells me it's purpose.